cluster/saltbase/salt/kube-dns/kubedns-rc.yaml.in

```diff
@@ -54,13 +54,21 @@ spec:
         args:
         # command = "/kube-dns"
         - --domain={{ pillar['dns_domain'] }}.
+        - --dns-port=10053
         ports:
         - containerPort: 53
```

Comment:
you should update the ports info to :10053 here and duplicate this into dnsmasq on :53